src/components/Onboarding/OnboardingPage2.tsx

```diff
@@ -3,39 +3,45 @@ import { Image, Text, TextInput, View } from 'react-native';
 import { TouchableOpacity } from 'react-native-gesture-handler';
 import * as Progress from 'react-native-progress';
 import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+import { useNavigation } from '@react-navigation/core';
 import styles from '../../styles/OnboardingStyles';
+import { CreateHomeInput } from '../../API';
 
 interface Props {
-  setPage(page: string): void;
   setHomeData({}): void;
-  homeData: Record<string, string>;
+  homeData: CreateHomeInput;
 }
 
-const Page2: React.FC<Props> = ({ setPage, setHomeData, homeData }) => {
-  const [squareFeet, setSquareFeet] = useState(homeData.squareFeet);
-  const [outdoor, setOutdoor] = useState(homeData.outdoor);
-  const [bedrooms, setBedrooms] = useState(homeData.bedrooms);
-  const [bathrooms, setBathrooms] = useState(homeData.bathrooms);
+const Page2: React.FC<Props> = ({ setHomeData, homeData }) => {
+  const [livableArea, setSquareFeet] = useState(String(homeData.livableArea));
+  const [outDoorArea, setOutdoor] = useState(String(homeData.outDoorArea));
+  const [bedroomCount, setBedrooms] = useState(String(homeData.bedroomCount));
+  const [bathroomCount, setBathrooms] = useState(
+    String(homeData.bathroomCount),
+  );
```

Comment:
nit: `number.toString` is preferrable to `String()` since `String(number)` silently fails if the number is null/undefined, while `number.toString` will explicitly throw an error. See [String vs toString](https://stackoverflow.com/questions/3945202/whats-the-difference-between-stringvalue-vs-value-tostring)

Reply:
I've tried this and I kept getting `Object is possibly 'null' or 'undefined'` so I opted for `String()`. Any suggestions on how to fix this error?

Reply:
I think [optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) (`number?.toString()`) will work, but I don't really have a strong opinion about this vs `String()`, so either way is fine with me

Reply:
I tried that as well but the `parseInt()` function I use later gives an error since it can't have an optional. I think we can go with `String()` for now since it seems to work fine.